### PR TITLE
Update cisco-sla.inc.php

### PR DESCRIPTION
**Pull request by tamirhad**
added status as field
added SLA name(as configured in the router) as tag
influxdb query is much convenient now

Please give a short description what your pull request is for

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/polling/cisco-sla.inc.php [hunk 1]
```diff
@@ -42,12 +43,13 @@
 
         $fields = array(
             'rtt' => $rtt,
+            'opstatus' => $opstatus,
```
Comment (murrant) [inline includes/polling/cisco-sla.inc.php:46]: Not defined for RRD (and rrd are immutable)

Comment (murrant) [reply includes/polling/cisco-sla.inc.php:46]: You'll need to create an additional RRD file to hold this new data.

Comment (laf) [reply includes/polling/cisco-sla.inc.php:46]: @tamirhad Any response to this?

**Comment by Jellyfrog** [comment]
Thank you for your contribution but we're closing this due to inactivity.  
Feel free to reopen if you plan on finishing it!

**Pull request closed without merging**